src/SprykerAbstractRule.php

```diff
@@ -39,4 +41,20 @@ protected function getModuleName(string $namespacedName): ?string
 
         return $matches['moduleName'];
     }
+
+    /**
+     * @param \PHPMD\Node\ClassNode $node
+     *
+     * @return \PDepend\Source\AST\AbstractASTClassOrInterface|null
+     */
+    protected function findFirstClassInterface(ClassNode $node): ?AbstractASTClassOrInterface
+    {
+        $classNodeInterfaces = $node->getInterfaces();
+
+        if (!$classNodeInterfaces->count()) {
+            return null;
```

Comment:
Actually we could add a violation here and this will cover the undescribed exceptions.

Reply:
Are there any actual occurrences in our current code? If so, can you let me know please.